src/components/Event/index.tsx

```diff
@@ -0,0 +1,71 @@
+import VolunteerList from '@/app/components/VolunteerList';
+import { EventResponse } from '@/types/dataModel/event';
+import { VolunteerResponse } from '@/types/dataModel/volunteer';
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+
+interface EventProps {
+  event: EventResponse;
+  volunteers: VolunteerResponse[];
+}
+
+export default function Event({
+  event,
+  volunteers,
+}: EventProps): React.ReactElement {
+  const eventNameLength = event.name.length;
+  const eventNameFontSize = eventNameLength > 20 ? 'h4' : 'h3';
```

Comment:
id prefer we dont have differences in size based on the text length, if anything we should be changing size based on the viewport (mobile, desktop etc) but since were only building for mobile i wouldnt worry about that rn. in summary just use one size or another

Reply:
will do